govuk-one-login/tech-docs
# Integrating third party platform

## Why

Need documentation for users integrating using third-party platforms. 

## What

New documentation gives guidance on third-party platforms and client_secret_post token authentication method.

## Technical writer support

This has been through tech writer review. 

## How to review

- check out the `integrating-third-party-platform` branch
- build the content using the new `./build-with-docker.sh` script and ensure that the html-proofer step at the end of the build runs cleanly and does not generate any errors that halt the build
- preview the content using `./preview-with-docker.sh` to ensure that the new content works in the context of the overall site

## Changelog

Changelog has been updated. 

## Confirm

- [y] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, elsewhere in these docs)
- [y] Where there is any overlap I have updated or opened a PR for corresponding changes
